cpp/data/sql_updates/ub_tools.18

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE metadata_presence_tracer MODIFY COLUMN metadata_field_name CHAR(3);
+ALTER TABLE metadata_field_name ADD COLUMN record_type ENUM('regular_article', 'review') DEFAULT 'regular_article';
```

Comment:
- TABLE should be `metadata_presence_tracer` and not `metadata_field_name` (2*)
- `metadata_field_name` and `record_type` should both contain `NOT NULL`